go/pkg/client/cas.go

```diff
@@ -6,8 +6,8 @@ import (
 	"os"
 	"path/filepath"
 	"sort"
-	"strings"
 
+	ctxmd "github.com/bazelbuild/remote-apis-sdks/go/pkg/contextmd"
```

Comment:
Do we need ctxmd? Can we directly use the package name?

Reply:
Hmm, I prefer the shorter form, but sure.